docs/modules/ROOT/pages/explanations/decisions/postgres-monitoring.adoc

```diff
@@ -0,0 +1,109 @@
+= PostgreSQL Managed by VSHN Monitoring
+
+== Problem
+
+We need to provide PostgreSQL Managed by VSHN monitoring checks:
```

Comment:
Should we mention and link to where this requirement comes from? Aka the SLI/SLO/SLA.